stock_reserve/migrations/8.0.2.0.0/post-migration.py

```diff
@@ -16,8 +14,19 @@
 #
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
-from . import stock_reserve
-from . import product
+
+def migrate(cr, installed_version):
```

Comment:
The migration script intends to update the reserved locations already created in previous versions. The post-init hook is now going to create extra reserved locations for additional companies.